pkg/types/azure/metadata.go

```diff
@@ -0,0 +1,12 @@
+package azure
+
+// Metadata contains AWS metadata (e.g. for uninstalling the cluster).
+type Metadata struct {
+	Region string `json:"region"`
+
+	// Identifier holds a slice of filter maps.  The maps hold the
+	// key/value pairs for the tags we will be matching against.  A
+	// resource matches the map if all of the key/value pairs are in its
+	// tags.  A resource matches Identifier if it matches any of the maps.
+	Identifier []map[string]string `json:"identifier"`
```

Comment:
resources in the Azure are probably most grouped by the resource group. So this map of identifies doesn't make a lot of sense.

Metadata should include information about the cluster that helps identify the cluster created by the installer.

Reply:
Correct. As the destroy has not been implemented yet, I didn't look into this. removing for now then.

Reply:
addressed